src/EditorFeatures/CSharpTest/SplitOrMergeIfStatements/MergeConsecutiveIfStatementsTests_ElseIf_WithNext.cs

```diff
@@ -0,0 +1,444 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.SplitOrMergeIfStatements
+{
+    public sealed partial class MergeConsecutiveIfStatementsTests
```

Comment:
Yes, the idea is that WithPrevious only tests with previous and WithNext only tests merging with next. In reality WithPrevious constains a lot more tests because that is what was originally implemented.

You're right that I'm breaking that rule a little. There are 3 tests at the bottom in this file that I think should really be next to each other, even though 1 of them only merges with previous and one of them merges with both (so that one doesn't really fit into either file anyway). Do you have an idea on how to organize this better?

Reply:
Maybe just have the ones that test both in a separate place? I think I'd also suggest not using partial types here and just putting the "WithWhatever" in the actual class name. My worry is if we're also looking at a test run output that just lists the fully qualified method name as a test failure, it won't be obvious which file it was in.

Reply:
> I think I'd also suggest not using partial types here and just putting the "WithWhatever" in the actual class name. My worry is if we're also looking at a test run output that just lists the fully qualified method name as a test failure, it won't be obvious which file it was in.

I'm on the fence about that one. I like that there's just 4 groups in the test explorer for each refactoring and I can easily run all tests for a certain refactoring:
![image](https://user-images.githubusercontent.com/11444821/51278303-836c7980-19da-11e9-92a3-d089b7774b20.png)

Reply:
➡️ Fixed in 59a97eed39a41301bce66315cb59771abcf0bc44